libh2o-evloop.pc.in

```diff
@@ -7,5 +7,5 @@ Name: libh2o-evloop
 Description: An optimized HTTP/1.x & HTTP/2 library
 URL: https://h2o.examp1e.net/
 Version: @LIBRARY_VERSION_MAJOR@.@LIBRARY_VERSION_MINOR@.@LIBRARY_VERSION_PATCH@
-Libs: -L${libdir} -lh2o-evloop
+Libs: -L${libdir} -lh2o-evloop -lwslay
```

Comment:
Thank you for the PR.

As our support for websocket is conditional (only enabled when wslay is detected), I think this should be `@WSLAY_LIBRARIES@`.

Would you mind changing to that and see if it works?

Reply:
Sure. I'll have a look when I'm home 👍

Reply:
The block where that's set in CMakeLists seems a bit odd, since it would only be set if the library -wasn't- found? Is that the reverse of the behaviour that we want? Same goes for the block above handling libuv.

```
IF (PKG_CONFIG_FOUND)
    PKG_CHECK_MODULES(WSLAY libwslay)
    IF (WSLAY_FOUND)
        INCLUDE_DIRECTORIES(${WSLAY_INCLUDE_DIRS})
        LINK_DIRECTORIES(${WSLAY_LIBRARY_DIRS})
    ENDIF (WSLAY_FOUND)
ENDIF (PKG_CONFIG_FOUND)
IF (NOT WSLAY_FOUND)
    FIND_PACKAGE(Wslay)
    IF (WSLAY_FOUND)
        INCLUDE_DIRECTORIES(${WSLAY_INCLUDE_DIR})
    ENDIF (WSLAY_FOUND)
ENDIF (NOT WSLAY_FOUND)
IF (NOT WSLAY_FOUND)
    SET(WSLAY_LIBRARIES -lwslay)
ENDIF (NOT WSLAY_FOUND)
```

Reply:
> The block where that's set in CMakeLists seems a bit odd, since it would only be set if the library -wasn't- found?

`WSLAY_LIBRARIES` is set by `PKG_CHECK_MODULES` or `FIND_PACKAGE` when libwslay is found. When it is not found, we set it to `-lwslay`, to avoid CMake complaining that we are using an undefined variable. As avoiding warnings is the only reason for setting the variable, it could be something else I think (e.g. `-lwslay-not-found`).